pyscriptjs/src/components/base.ts

```diff
@@ -1,11 +1,18 @@
-import { pyodideLoaded } from '../stores';
+import { runtimeLoaded } from '../stores';
 import { guidGenerator, addClasses, removeClasses } from '../utils';
-import type { PyodideInterface } from '../pyodide';
-// Premise used to connect to the first available pyodide interpreter
-let runtime: PyodideInterface;
+
+/*
+All references to `PyodideInterface` have been replaced with
+`Runtime` which has the available methods of running code asynchronously,
+getting the globals symbol table, etc. which each runtime is responsible
+for implementing on its own.
+*/
+
+import type { Runtime } from '../runtime';
+let runtime: Runtime;
```

Comment:
This is a good first step towards the direction we want to go (a generic runtime). 

On thing that will need to follow (in a follow up PR, not now! let's get it working first) is replacing this with a data structure that can allow use to support multiple runtimes.

Reply:
Yeah, we can do this later, thanks!